drape/font_texture.cpp

```diff
@@ -179,11 +155,12 @@ ref_ptr<Texture::ResourceInfo> GlyphIndex::MapResource(GlyphKey const & key, boo
 
     return nullptr;
   }
-  generationData.m_glyph = glyph;
-  generationData.m_rect = r;
 
   auto res = m_index.emplace(key, GlyphInfo(m_packer.MapTextureCoords(r), glyph.m_metrics));
   ASSERT(res.second, ());
+
+  m_pendingNodes.emplace_back(r, std::move(glyph));
```

Comment:
IDK, do we still need this?

Reply:
Yes, it is used in GlyphIndex::UploadResources to upload rendered glyphs to the texture. Without this call all SDF glyph rendering is useless )

Reply:
I mean all m_pendingNodes routine.
It was needed with previous logic when glyph rendering was _delayed_ (pending).
You did immediate glyph rendering on creation.
So the next logical step is to remove m_pendingNodes at all.

Reply:
@vng that's not feasible at the moment. Glyphs are uploaded from another DrapeFronend thread which uses stored graphics context to upload data to textures. That context is not available here in this thread. So m_pendingNodes is just a queue to pass glyphs to another thread.